tests/phpunit/unit-tests/user/class-llms-test-student.php

```diff
@@ -100,10 +100,13 @@ public function test_completion_incompletion() {
 	}
 
 	/**
-	 * Test whether a user is_enrolled() in a course or membership
+	 * Test whether a user is_enrolled() in a course or membership.
+	 *
+	 * @since 3.5.0 Introduced.
```

Comment:
```suggestion
	 * @since 3.5.0
```

Introduction is implied with the first tag in the list